modules/snooker/src/main/ts/ephox/snooker/queries/TableTargets.ts

```diff
@@ -19,9 +18,9 @@ const noMenu = (cell: Element<HTMLTableCellElement | HTMLTableCaptionElement>):
   selection: Fun.constant([ cell ])
 });
 
-const forMenu = (selections: Selections, table: Element<HTMLTableElement>, cell: Element<HTMLTableCellElement>): RunOperation.CombinedTargets => ({
+const forMenu = (selections: Selections, table: Element<HTMLTableElement>, cell: Element<HTMLTableCellElement>, firstSelectedSelector: string, lastSelectedSelector: string): RunOperation.CombinedTargets => ({
```

Comment:
Would it make more sense to have this be an object called `selectors`? That'd likely work better if you make `Ephemera` an interface/object.